content/06.model-complexity.md

```diff
@@ -11,30 +11,31 @@ All of these contribute to low signal to noise ratio in rare disease datasets.
 Thus, applying ML to rare disease data without addressing the aforementioned shortcomings may lead to models that have low reproducibility or are hard to interpret.
 
 Class imbalance in datasets can be addressed using decision tree-based ensemble learning methods (e.g., random forests) (Figure[@fig:3]a).
-Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10/btzfh6]. 
+Random forests use resampling (with replacement) based techniques to form a consensus about the important predictive features identified by the decision trees [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10/btzfh6]. [TODO: consider re-evaluating these references now that we're not explicitly talking about bagging?]
```

Comment:
A lot of these references are about bagging specifically. Do we want to take them out/reconsider now that this section has been modified?